XRDXRFutils/gaussnewton.py

```diff
@@ -41,11 +41,10 @@ def __init__(self, phase, spectrum, min_theta = 0, max_theta = 53, min_intensity
         self.mu, self.I = self.get_theta(min_theta = min_theta, max_theta = max_theta, min_intensity = min_intensity)
```

Comment:
Maybe directly?
self.mu[newaxis, :]

Reply:
Good suggestion. Done in [0df4b7e](https://github.com/zpreisler/XRDXRFutils/pull/3/commits/0df4b7e3ec9b3e812f8ba322016de9396ccd7e9d).